Hide `vec` and `vapp`

bishboria opened the pull request:
Vector tooling has been moved to `CS410-Vec.agda` but `vec` and `vapp` weren't removed from this file.

They are useful to work out, so hiding them from being imported instead of removing.

jmchapman commented:
You could hide them instead, eg. open Import CS410-Prelude hiding (vec; vapp) or something like that.

bishboria commented:
Of course! Much nicer. Will hide the prelude versions instead and update.

bishboria committed:
Hide `vec` and `vapp` from `CS410-Vec`

Vector tooling has been moved to CS410-Vec.agda but vec and vapp weren't removed from this file.

They are useful to work out, so hiding them from being imported instead of removing.
